File: FizzBuzz.java
public class FizzBuzz{public static void main(String[] args){
int n=1;
for(;;){n+=1;String x="";
	if(n%3==0)x+="Fizz";
	if(n%5==0)x+="Buzz";
	if(x=="")System.out.print(n);
	System.out.println(x);}}}

File: FizzBuzz.pas
program FizzBuzz;
var
i: integer;
x: boolean;
begin
i:=1;
while i>0 do
	begin
	x:=true;
	if i mod 3 = 0 then begin x:=false;write('Fizz') end;
	if i mod 5 = 0 then begin x:=false;write('Buzz') end;
	if x then write(i);
	writeln();i+=1;
	end;
end.
